test/chainsaw/integration/restartable-app/00-install.yaml

```diff
@@ -0,0 +1,333 @@
+apiVersion: lifecycle.keptn.sh/v1beta1
+kind: KeptnApp
+metadata:
+  name: podtato-head
+spec:
+  version: "1.3"
+  workloads:
+    - name: podtato-head-left-arm
+      version: 0.1.0
+    - name: podtato-head-left-leg
+      version: 0.1.0
+    - name: podtato-head-entry
+      version: 0.1.0
+    - name: podtato-head-right-arm
+      version: 0.1.0
+    - name: podtato-head-right-leg
+      version: 0.1.0
+    - name: podtato-head-hat
+      version: 0.1.0
+  postDeploymentTasks:
+    - post-deployment-hello
+  preDeploymentTasks:
+    - post-deployment-hello
```

Comment:
This is not valid anymore, should we keep it there ? @RealAnna

Reply:
yes, v1beta1 does not have pre/post deployment tasks/evaluations

Reply:
🤔 
```
| 09:25:27 | restartable-app | step-00  | CREATE    | OK    | lifecycle.keptn.sh/v1beta1/KeptnApp @ chainsaw-fleet-rooster/podtato-head
```

Reply:
the fields are probably just ignored in the test